06week/higherOrder.js

```diff
@@ -3,19 +3,50 @@
 const assert = require('assert');
 
 function forEach(arr, callback) {
```

Comment:
Couple things here. `.forEach()` doesn't return anything, so no need for lines 7/13. Also, the callback acts on each item in the array, so make sure to pass each in!